lava_test_plans/projects/lkft-android/devices/hikey

```diff
@@ -0,0 +1 @@
+hi6220-hikey-r2
```

Comment:
please drop the "hikey" link since there isn't any devices in the lab called that.